packages/cid-verifier/test/verification.spec.js

```diff
@@ -43,105 +39,74 @@ test.before(async (t) => {
       })
     }
   })
-  const denylistKv = await mf.getKVNamespace('DENYLIST')
-  await denylistKv.put(await toDenyListAnchor(cidInDenyList), JSON.stringify({ status: 410, reason: 'bad' }))
-  await denylistKv.put(await toDenyListAnchor(cidInDenyListBlockedForLeganReasons), JSON.stringify({ status: 451, reason: 'blocked for legal reasons' }))
 })
 
-test('GET /denylist handles cids in DENYLIST', async (t) => {
+test('GET /:cid handles invalid cid', async (t) => {
   const { mf } = t.context
-  const response = await mf.dispatchFetch(`http://localhost:8787/denylist?cid=${cidInDenyList}`)
-  t.is(await response.text(), 'MALWARE DETECTED')
-  t.is(response.status, 403)
-})
-
-test('GET /denylist handles cids in DENYLIST blocked for legal reasons', async (t) => {
-  const { mf } = t.context
-  const response = await mf.dispatchFetch(`http://localhost:8787/denylist?cid=${cidInDenyListBlockedForLeganReasons}`)
-  t.is(await response.text(), 'BLOCKED FOR LEGAL REASONS')
-  t.is(response.status, 451)
-})
-
-test('GET /denylist handles no cid', async (t) => {
-  const { mf } = t.context
-  const response = await mf.dispatchFetch('http://localhost:8787/denylist?')
-  t.is(await response.text(), 'cid is a required query param')
-  t.is(response.status, 400)
-})
-
-test('GET /denylist handles invalid cid', async (t) => {
-  const { mf } = t.context
-  const response = await mf.dispatchFetch('http://localhost:8787/denylist?cid=invalid')
-  t.is(await response.text(), 'cid query param is invalid')
+  const response = await mf.dispatchFetch('http://localhost:8787/invalid')
+  t.is(await response.text(), 'cid path param is invalid')
   t.is(response.status, 400)
 })
 
-test('GET /denylist handles no results', async (t) => {
+test('GET /:cid handles no results', async (t) => {
   const { mf } = t.context
-  const response = await mf.dispatchFetch(`http://localhost:8787/denylist?cid=${emptyCid}`)
+  const response = await mf.dispatchFetch(`http://localhost:8787/${emptyCid}`)
   t.is(await response.text(), '')
   t.is(response.status, 204)
 })
 
-test('GET /denylist handles pending results', async (t) => {
+test('GET /:cid handles pending results', async (t) => {
   const { mf } = t.context
-  const response = await mf.dispatchFetch(`http://localhost:8787/denylist?cid=${pendingCid}`)
+  const response = await mf.dispatchFetch(`http://localhost:8787/${pendingCid}`)
   t.is(await response.text(), '')
   t.is(response.status, 204)
 })
 
-test('GET /denylist handles successful results', async (t) => {
+test('GET /:cid handles successful results', async (t) => {
   const { mf } = t.context
-  const response = await mf.dispatchFetch(`http://localhost:8787/denylist?cid=${notMalwareCid}`)
+  const response = await mf.dispatchFetch(`http://localhost:8787/${notMalwareCid}`)
   t.is(await response.text(), '')
   t.is(response.status, 204)
 })
 
```

Comment:
Can we add test for GET malicious CID `malwareCid` ? I think is the only flow missing :)